<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerOpacityDialog</class>
 <widget class="QDialog" name="LayerOpacityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer / Keyframe Opacity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labLayerInfo">
       <property name="text">
        <string>Layer: </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutSlider">
     <item>
      <widget class="QSlider" name="chooseOpacitySlider">
       <property name="toolTip">
        <string>% transparency</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="chooseOpacitySpinBox">
       <property name="toolTip">
        <string>% transparency</string>
       </property>
       <property name="suffix">
        <string> %</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="singleStep">
        <double>0.200000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOpacity">
     <property name="title">
      <string>Set opacity for:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbActiveKeyframe">
        <property name="text">
         <string>Active keyframe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbSelectedKeyframes">
        <property name="text">
         <string>Selected keyframe(s)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbActiveLayer">
        <property name="text">
         <string>Layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFade">
     <property name="title">
      <string>Fade in / Fade out</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnFadeIn">
        <property name="toolTip">
         <string>Fade in over selcted keyframes</string>
        </property>
        <property name="text">
         <string>Fade in</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFadeOut">
        <property name="toolTip">
         <string>Fade out over selected keyframes</string>
        </property>
        <property name="text">
         <string>Fade out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutBtnClose">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labInfo">
     <property name="text">
      <string>Be aware that opacity changes are made in the rendering, and will not change your artwork.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
